docs/README.md

```diff
@@ -0,0 +1,107 @@
+# Octokit Requirements
+
+> Provide a first class experience for platform developers in their preferred programming languages
+
+Octokit strives to offer complete and officially supported SDKs, quick starts, code samples, and copy/paste-able examples for the GitHub Platform in the most popular languages.
+
+Each Octokit will work towards meeting these requirements:
+
+|                                 | JavaScript | Ruby | .NET |
+| ------------------------------- | ---------- | ---- | ---- |
+| [Complete](#complete)           | 🌔         | 🌔   | 🌔   |
+| → REST                          | ✅         | ✅   | 🌔   |
+| → GraphQL                       | 🌔         | 🌑   | 🌔   |
+| → GitHub Apps                   | 🌔         | 🌔   | ✅   |
+| → OAuth                         | 🌔         | ✅   | ✅   |
+| → Webhooks                      | ✅         | 🌑   | 🌔   |
+| [Decomposable](#decomposable)   | ✅         | 🌑   | 🌑   |
```

Comment:
> Each feature is available both as individual packages and **one monolithic meta-package** that includes all components pre-wired.

@gr2m I think this should be 🌔 since the `octokit.js` meta-package has been released yet.